vcomponent/profiles/test_sink.yaml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
should we keep testing profiles (used to automate the module checks), independent of the real profiles that will be used for the platform testing? Am wondering if we should be under /src/profiles/blah.yaml rather than in vcomponent/profiles?

what do you think?